internal/proxy/proxy.go

```diff
@@ -582,6 +584,26 @@ func (p *Proxy) createRequestHash(url string, headers string) uint32 {
 	return h.Sum32()
 }
 
+func (p *Proxy) getUserWithRetry(repositoryCloneURL string, logEntry *log.Entry, retry int) (clients.Namespace, error) {
+	var namespace clients.Namespace
```

Comment:
Well actually it is close to `getUser` which is what I want since this method is a wrapper of `getUser`.

Reply:
I have seen both usages `err` and `error` but I am not a golang expert so not sure about it.

Reply:
@sthaha is more go-way imho but i am fine with both,

Reply:
Wouldn't it be `err` instead of `error` to avoid confusing variable name with type `error`: https://golang.org/pkg/builtin/#error .. e.g.
```go
var err, float32 error
```
is a valid variable declaration albeit confusing.

Reply:
the last return bit of the function declaration is a type in golang not a variable here,

you are just saying you are going to return something of an 'error' type,

Reply:
aah I mean the line below .. 
```
var error error
``` 
not the function signature :)